### PR TITLE
[guilib] Add randomize tag for fadelabels

### PR DESCRIPTION
**Pull request by BigNoid**
This utilizes the `randomize` tag for fadelabel controls so it randomly sorts the label tags. Defaults to false.

@mkortstiege @ronie

### CODE REVIEW

**Comment by mkortstiege** [comment]
Looks good to me. Could you please add the API bump for the legacy/python part?

**Comment by BigNoid** [comment]
@mkortstiege sure, python API bumped

File: xbmc/guilib/GUIFadeLabelControl.cpp [hunk 1]
```diff
@@ -62,6 +64,8 @@ void CGUIFadeLabelControl::SetInfo(const vector<CGUIInfoLabel> &infoLabels)
 {
   m_lastLabel = -1;
   m_infoLabels = infoLabels;
+  if (m_randomized)
+    random_shuffle(m_infoLabels.begin(), m_infoLabels.end());
```
Comment (mkortstiege) [inline xbmc/guilib/GUIFadeLabelControl.cpp]: Please change to std::random_:shuffle. After that is's good to go (now really).

**Comment by BigNoid** [comment]
@mkortstiege done, thx for review.

**Comment by ronie** [comment]
nice addition. +1 for sure.

**Comment by BigNoid** [comment]
@ronie great :) 
@mkortstiege Added a cosmetics commit to remove ```namespace std``` from GUIFadeLabelControl.cpp. If that one is okay too, can you call a jenkins build and merge?

**Comment by mkortstiege** [comment]
You just triggered it ;)

**Comment by BigNoid** [comment]
Yeah i noticed. Didnt think I could since I dont have write access. Ah well
:P

**Comment by hudokkow** [comment]
jenkins build this please

**Comment by mkortstiege** [comment]
@BigNoid please rebase. The namespace commit can be ditched since @MartijnKaijser merged my bigger cleanup PR already.

**Comment by BigNoid** [comment]
rebased

**Comment by mkortstiege** [comment]
> rebased

Thanks!

**Pull request merged**